README.md

```diff
@@ -1,151 +1,150 @@
+</a>
+
+<p align="center">
+  <a href="https://youtu.be/mhszO6kZSbI">
+    <img src="https://img.youtube.com/vi/mhszO6kZSbI/maxresdefault.jpg" alt="YourBench Demo Video" width="600" style="border-radius: 10px; box-shadow: 0 4px 8px rgba(0,0,0,0.1);" />
+    <br>
+    <img src="https://img.shields.io/badge/Watch%20Demo-YouTube-red?style=for-the-badge&logo=youtube" alt="Watch Demo on YouTube">
+    <br>
+    <em>Watch our 3-minute demo of the YourBench pipeline</em>
+  </a>
+</p>
 
-**Dynamic Evaluation Set Generation for LLM Benchmarking [NAACL '25*]*
+</div>
 
-[![Python 3.12+](https://img.shields.io/badge/python-3.12+-blue.svg)](https://www.python.org/downloads/)
-[![Code style: ruff](https://img.shields.io/badge/code%20style-ruff-000000.svg)](https://github.com/astral-sh/ruff)
-[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-[![🤗 Hugging Face](https://img.shields.io/badge/huggingface-datasets-yellow)](https://huggingface.co/docs/datasets)
+---
 
-</div>
+> **YourBench** is an open-source framework for generating domain-specific benchmarks in a zero-shot manner, inspired by modern software testing practices. It aims to keep your large language models on their toes—even as new data sources, domains, and knowledge demands evolve.
 
-## 🌟 Overview
+**Highlights**:
+- **Dynamic Benchmark Generation**: Produce diverse, up-to-date questions from real-world source documents (PDF, Word, HTML, even multimedia).
+- **Scalable & Structured**: Seamlessly handles ingestion, summarization, and multi-hop chunking for large or specialized datasets.
+- **Zero-Shot Focus**: Emulates real-world usage scenarios by creating fresh tasks that guard against memorized knowledge.
+- **Extensible**: Out-of-the-box pipeline stages (ingestion, summarization, question generation), plus an easy plugin mechanism to accommodate custom models or domain constraints.
 
-Yourbench is a powerful framework for dynamically generating evaluation sets from source documents. It addresses the limitations of static benchmarks and benchmark saturation by creating diverse, contextually-rich questions tailored to specific educational levels.
+---
 
-### 🔄 Process Flow
+## Quick Start (Alpha)
 
-![Process Flow](static/images/process-figure.png)
+```bash
+# 1. Clone the repo
+git clone https://github.com/huggingface/yourbench.git
+cd yourbench
+
+# Use uv to install the dependencies
+# pip install uv # if you do not have uv already
+uv venv
+source .venv/bin/activate
+uv sync
+uv pip install -e .
+
+# 3. Get a key from https://openrouter.ai/ and add it to the .env file (or make your own config with a different model!)
+touch .env
+echo "HF_TOKEN=<your_huggingface_token>" >> .env
+
+# 4. Run the pipeline with an example config
+yourbench run --config configs/example.yaml
+```
 
-## ✨ Features
+You can also launch a minimal Gradio UI by including `--gui`. 
```

Comment:
Is it working now ? Maybe point to the space instead

Reply:
i was planning on integrating the space logic into the codebase. but right now, no, it doesn't work :(